### DISCUSSION TITLE
Converting binary data that to RGBA and supporting also HDR fomrat

### DISCUSSION THREAD

**Discussion by AndreAhmed** [nothings/stb#1475] [Q&A]
I'm running into a situation that I have bin. files that should contain binary data, and need to encode them into RGB Images, there is another issue that some of these files are HDR files too. I would like to know a way on how to convert that binary files into acceptable format that can be encoded so that I can display them. The display of them should also as input, to have RGBA data. I have looked through stbi library, but it can't load binary files and encode them.

**Comment by rygorous** [comment]
"Binary file" is not a format, it just means that the data contained is not ASCII text, which is to say, just a bag of bytes. That's not something stb_image can help with; it's a loader for a certain set of well-known image formats. It can load these exact formats but if it's not one of those, you need to figure out (or ask whomever you got that data from) what the format is and load it yourself.
